doc/mgr/rgw.rst

```diff
@@ -139,3 +139,33 @@ Join an existing realm by creating a new secondary zone (using the realm token)
   ceph rgw admin [*]
 
 RGW admin command
+
+Automatic usage log trimming
+----------------------------
+
```

Comment:
@tobias-urdin I think we have to add some note to warn the users that this in fact is not fully automated as in case the user add/remove realms the list must be updated manually. WDYT?

Reply:
I can update documentation to make that more clear!